{"uuid": "cb5727dc-58c9-4977-a562-ac5e0fd8ae2d", "children": ["adba2bf8-cfe6-4cea-b427-e5a81c02f5cb"], "befores": [{"name": "update_food_init", "status": "broken", "statusDetails": {"message": "IndexError: list index out of range\n", "trace": "  File \"C:\\Users\\Administrator\\PycharmProjects\\apiAutoTest\\venv\\lib\\site-packages\\pluggy\\_callers.py\", line 77, in _multicall\n    res = hook_impl.function(*args)\n  File \"C:\\Users\\Administrator\\PycharmProjects\\apiAutoTest\\venv\\lib\\site-packages\\_pytest\\fixtures.py\", line 1123, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"C:\\Users\\Administrator\\PycharmProjects\\apiAutoTest\\venv\\lib\\site-packages\\_pytest\\fixtures.py\", line 902, in call_fixture_func\n    fixture_result = fixturefunc(**kwargs)\n  File \"C:\\DeliverySystem完整代码\\testCase\\conftest.py\", line 72, in update_food_init\n    idMenu = food_init.query({\"page\": 1, \"limit\": 1})['data']['records'][0]['category_id']\n"}, "start": 1704554556146, "stop": 1704554556358}], "start": 1704554556146, "stop": 1704554556371}